rotkehlchen/chain/substrate/manager.py

```diff
@@ -415,7 +415,12 @@ def _get_node_interface(self, endpoint: str) -> SubstrateInterface:
             log.error(message)
             raise RemoteError(message) from e
         except (FileNotFoundError, ValueError, TypeError) as e:
-            raise AttributeError('Invalid SubstrateInterface instantiation') from e
+            message = (
```

Comment:
So @vnavascues question here. From what I understand this is general a "should not happen case"? Raising an AttributeError here and not handling it anywhere above is not good. Unless it's an assertion error.

So I turned it into a remote error and a log entry that we had an unknown exception. Could also be turned to log entry + assertion error if it should never happen. But as we saw it happened already once in this bug.